### PR TITLE
Faq esc key#693

### PR DESCRIPTION
**Pull request by praneetharra**
Here is the edit for issue #693

### CODE REVIEW

**Comment by praneetharra** [comment]
@dogi @Loshma93 @mappuji Review the following pull request.

**Comment by praneetharra** [comment]
@waryhermit I am not sure why the second change popped up in this pull request, it was already merged here. https://github.com/open-learning-exchange/open-learning-exchange.github.io/pull/691

**Comment by waryhermit** [comment]
@praneetharra You probably did not sync your master branch or `Faq_Esc_Key#693` after those changes were merged. You can sync now or checkout that file from the upstream.

File: pages/htmlresources.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Documentation helps to understand overall progress of project. Visit following l
 
 Prerequisites:
 
-* [Couchapp](#couchapp-installation)
+* [Couchapp](htmlresources.md#Couchapp_Installation)
```
Comment (dogi) [inline pages/htmlresources.md]: @praneetharra I think you branch off of one of your feature branches
that is why you have this file too much ...

**Comment by praneetharra** [comment]
@dogi @Loshma93 @waryhermit I made the necessary changes. I guess everything should be fine now. Let me know any further changes.

**Pull request merged**